app/services/courses/query.rb

```diff
@@ -161,6 +162,24 @@ def funding_scope
       @scope.where(funding: params[:funding])
     end
 
+    def start_date_scope
+      return @scope if params[:start_date].blank?
+
+      @applied_scopes[:start_date] = params[:start_date]
+
+      current_recruitment_cycle_year = Settings.current_recruitment_cycle_year
```

Comment:
I chosen `Settings.current_recruitment_cycle_year` because I was reluctanly to hit the database here to only get the year.